StatsdClient/StatsdClientExtensions.cs

```diff
@@ -34,7 +48,48 @@ public static void LogTiming(this IStatsd client, string name, long milliseconds
         /// <param name="value">The value for this gauge</param>
         public static void LogGauge(this IStatsd client, string name, int value)
         {
-            client.LogGaugeAsync(name, value).Wait();
+            try
+            {
```

Comment:
Can you revert this change, the aggregate exception should be boiled up as it can contain multiple exceptions.